tests/test_downloader.py

```diff
@@ -1,6 +1,7 @@
+import sys
+import os
+sys.path.append(os.getcwd())
```

Comment:
required?

Reply:
Receive the following error when running tests/test_downloader.py with
`from pyQuARC.code.downloader import Downloader`
```
Traceback (most recent call last):
  File "/Users/sam/Documents/GitHub/pyQuARC/tests/test_downloader.py", line 1, in <module>
    from pyQuARC.code.downloader import Downloader
ModuleNotFoundError: No module named 'pyQuARC'
```